grails-app/views/doiResolve/_phylolink.gsp

```diff
@@ -0,0 +1,218 @@
+<%@ page contentType="text/html;charset=UTF-8" %>

+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="layout" content="main"/>
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    %{-- Google Analytics --}%
+    <script>
+        window.ga = window.ga || function () {
+                    (ga.q = ga.q || []).push(arguments)
+                };
+        ga.l = +new Date;
+        ga('create', '${grailsApplication.config.googleAnalyticsId}', 'auto');
+        ga('send', 'pageview');
+    </script>
+    <script async src='//www.google-analytics.com/analytics.js'></script>
+    %{--End Google Analytics--}%
+
+    <asset:stylesheet src="doi.css"/>
+</head>
+
+<body>
+<ala:systemMessage/>
+
+<div class="col-sm-12 col-md-9 col-lg-9">
+    <div class="col-md-12" id="doiTitle">
+        <h2><a href="https://doi.org/${doi.doi}" type="button" class="doi"><span>DOI</span><span>${doi.doi}</span></a></h2>
+		<h3 class="heading-medium ${doi.active? '': 'text-muted'}">${doi.title}
+			<g:if test="${!doi.active}">
+				<small class="badge badge-secondary"><g:message code="doi.inactive"/></small>
+			</g:if>
+		</h3>
+    </div>
+
+    <div class="row">
+        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <div class="panel panel-default">
+                <div class="panel-body">
+                    <div class="word-limit break-word">
+                        <div class="row">
+                            <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+                                <div class="well">
+                                    <p style="margin-top: 4px; margin-bottom: 4px;">To access this resource, you can 
+                                        <a class="btn btn-default" href="${doi.applicationUrl}"
+										   style="margin-left: 4px; margin-top: -14px; margin-bottom: -12px;"


```

Comment:
There are `^M` chars in a few lines (also lines 63-68). Are they intentional? If not I think they should be removed.

Reply:
Changed EOL characters from Windows (CR LF) to Unix (LF) which has removed the ^M characters